R/requerimentos.R

```diff
@@ -54,12 +62,57 @@ fetch_related_requerimentos_camara <- function(prop_id, mark_deferimento = FALSE
       rename_table_to_underscore() %>%
       .assert_dataframe_completo(.COLNAMES_REQUERIMENTOS_CAMARA) %>%
       .coerce_types(.COLNAMES_REQUERIMENTOS_CAMARA, order_cols = FALSE) 
+    
+  }
+}
+
+#' @title Fetch related requerimentos in Senado 
+#' @description Returns a dataframe with data from requerimentos related to a given proposition in Senado
+#' @param prop_id ID of a Proposicao
+#' @param normalized True or False
+#' @return Dataframe
+#' @export
+fetch_related_requerimentos_senado <- function(prop_id, normalized = FALSE) {
```

Comment:
A lei de id 133943 não está retornando as relacionadas

Reply:
Idem para 129808

Reply:
O projeto de lei de id 127753 está retornando  `Error in FUN1(.) : could not find function "FUN1"`

Reply:
O projeto de lei de id 96813 está retornando um df vazio (sem colunas)

Reply:
Idem para 131906

Reply:
Mandei uma mensagem para os dados abertos do Senado, esperando resposta, pois aparece no site deles https://www25.senado.leg.br/web/atividade/materias/-/materia/133943 e na chamada dos dados abertos não aparece http://legis.senado.leg.br/dadosabertos/materia/133943